server/test/services/export/JsonExporterTest.java

```diff
@@ -174,6 +174,9 @@ private void testApplicationTopLevelAnswers(
         "$[" + resultIndex + "].applicant_id", application.getApplicant().id);
     resultAsserter.assertValueAtPath("$[" + resultIndex + "].application_id", application.id);
     resultAsserter.assertValueAtPath("$[" + resultIndex + "].language", "en-US");
+    resultAsserter.assertValueAtPath(
```

Comment:
issue: It's not clear from the TODO at the top of the file, but this test is being replaced soon.

suggestion: Instead, can we add a test in the style of `export_whenSubmitterIsTi_tiTopLevelFieldsAreSet()` for each of the `submission_status` types?

Reply:
Done